docs/products/flink/getting-started.rst

```diff
@@ -1,12 +1,74 @@
 Getting started
 ===============
 
-Aiven for Flink is available in the `Aiven console <https://console.aiven.io>`_.
+The first steps in using Aiven for Apache Flink are to create a service and a Flink job. You can do this in the Aiven web console or with the Aiven CLI.
 
-Choose the Apache Flink version, your cloud provider and location to deploy to, then choose which plan to use.
+In addition to the Flink service, you also need a service that you can use as a data source. This article outlines an example of using an Apache Kafka service as the data source.
```

Comment:
"In addition to the Flink service, you also need a service that you can use as a data source" I would move to "In addition to Flink, you also need one or more services that you can use as a data source and targets"

Reply:
Yep, that's now updated ;)